styles.js

```diff
@@ -0,0 +1,68 @@
+let books = [];
+
+function loadBooks() {
+  const storedBooks = localStorage.getItem('books');
+  if (storedBooks) {
+    books = JSON.parse(storedBooks);
+  } else {
+    books = [
+      { title: 'Pride and Prejudice', author: 'Jane Austen' },
+      { title: 'The Great Gatsby', author: 'F. Scott Fitzgerald' },
+      { title: 'To Kill a Mockingbird', author: 'Harper Lee' },
+      { title: 'The Lord of the Rings', author: 'J.R.R. Tolkien' },
+    ];
+  }
+}
+
+function saveBooks() {
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function displayBooks() {
+  const bookList = document.getElementById('book-list');
+  bookList.innerHTML = '';
+
+  for (let i = 0; i < books.length; i += 1) {
+    const book = books[i];
+    const booklistitem = document.createElement('li');
+    booklistitem.innerHTML = `<p>${book.title} <br> ${book.author}</p> <button class='remove-btn' data-index='${i}'>Remove</button> <hr>`;
+    bookList.appendChild(booklistitem);
+  }
+}
+
+function removeBook(title) {
+  const newBooks = books.filter((book) => book.title !== title);
```

Comment:
- [ OPTIONAL ] I highly recomend to add id and make it unique to your book list beside the title and the author and when you want to filter just filter by ID, because the title might be the same but the ID should be unique so when the user delete a book with the same title so both will be deleted so kindly fix this to improve from user experience (UX) as well✨💯🔥🔥